jobs/build/gen-assembly/Jenkinsfile

```diff
@@ -105,9 +105,6 @@ node {
         stage("initialize") {
             buildlib.registry_quay_dev_login()
             currentBuild.displayName += " - ${params.BUILD_VERSION} - ${params.ASSEMBLY_NAME}"
-            if (!params.IN_FLIGHT_PREV) {
```

Comment:
Let's change the parameter description to reflect that it will be auto-fetched if left empty